internal/boxcli/shell.go

```diff
@@ -36,6 +37,8 @@ func shellCmd() *cobra.Command {
 
 	command.Flags().BoolVar(
 		&flags.PrintEnv, "print-env", false, "print script to setup shell environment")
+	command.Flags().BoolVar(
+		&flags.Pure, "pure", false, "Creates an isolated shell without taking any variables from parent system.")
```

Comment:
what do you think about:

> without sharing any environment from the parent system.

?

Reply:
This is how nix described pure:


> --pure
If this flag is specified, the environment is almost entirely cleared before the interactive shell is started, so you get an environment that more closely corresponds to the “real” Nix build. A few variables, in particular HOME, USER and DISPLAY, are retained.

Reply:
Updated the flag description to a similar text.